apps/microservices-orders/orders-service/build.gradle

```diff
@@ -0,0 +1,71 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+  dependencies {
+    classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:0.21.0"
+  }
+}
+
+plugins {
+	id 'org.springframework.boot' version '2.3.4.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.10.RELEASE'
+	id 'java'
+  id "com.github.johnrengelman.shadow" version "6.1.0"
+  id 'com.google.cloud.tools.jib' version '2.6.0'
+}
+
+group = 'io.confluent.kafka-devops.microservices-orders'
+version = '10.0.0-SNAPSHOT'
+sourceCompatibility = '11'
+
+repositories {
+	mavenCentral()
+  maven {
+    url "https://packages.confluent.io/maven/"
+  }
+}
+
+apply plugin: "com.commercehub.gradle.plugin.avro"
+
+ext {
+	set('springCloudVersion', "Hoxton.SR8")
+}
+
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+  implementation 'org.springframework.boot:spring-boot-starter-actuator'
+  implementation 'org.springframework.boot:spring-boot-starter-webflux'
+	implementation 'org.apache.kafka:kafka-streams'
+	implementation 'org.springframework.cloud:spring-cloud-stream'
+	implementation 'org.springframework.cloud:spring-cloud-stream-binder-kafka-streams'
```

Comment:
@rspurgeon i don't see that you use any spring cloud stream bits. do you need this dependency? 

those can be enough 
```
implementation 'org.springframework.kafka:spring-kafka'
testImplementation 'org.apache.kafka:kafka-streams-test-utils'
```

Reply:
Remnant from previous attempts, thanks @gAmUssA removed in latest

Reply:
@gAmUssA I had to back out my changes.  When I remove some dependency declarations I get test failures.

Reply:
@gAmUssA I was able to see my error with the spring-cloud-stream deps and i've removed them, resolving this one.